back/spec/fixtures/locales/en.yml

```diff
@@ -64,5 +64,5 @@ en:
   xlsx_export:
     anonymous: "Anonymous"
   voting_method:
-    default_voting_term_singular: "vote"
```

Comment:
This should remain as lower case as this is going to be used within a sentence - eg 1 vote or 10 votes